features_and_tools/firely-server-ingest.rst

```diff
@@ -41,7 +41,13 @@ To install the tool, you first need to have .NET Core SDK v8.x installed on your
 
   dotnet tool install --global Firely.Server.Ingest
 
-The command above will install FSI from this `NuGet package <https://www.nuget.org/packages/Firely.Server.Ingest/>`_.
+You can update to the latest version using the following command:
+
+::
+
+  dotnet tool update --global Firely.Server.Ingest
+
+The commands above will install FSI from this `NuGet package <https://www.nuget.org/packages/Firely.Server.Ingest/>`_.
```

Comment:
> The commands above will install FSI from this `NuGet package <https://www.nuget.org/packages/Firely.Server.Ingest/>`_.

Shouldn't this sentence be below the install command?

Reply:
I don't know. I say:
> The command**S** above...

Would you like me to move it between the install and update commands?